boards/hail/src/main.rs

```diff
@@ -35,15 +35,15 @@ unsafe fn load_processes() -> &'static mut [Option<kernel::process::Process<'sta
         static _sapps: u8;
     }
 
-    const NUM_PROCS: usize = 2;
+    const NUM_PROCS: usize = 4;
```

Comment:
If only we could store the each Process struct in the space allocated for the Process... : ) But that sounds like an optimization for another day.